### PR TITLE
Add contact form with submission functionality

### PR DESCRIPTION
**Pull request by Julieanna97**
- Added a responsive contact form to the contact.html page
- Included fields for name, email, phone, address, subject, and description
- Implemented form submission with validation
- Added success message display on successful submission
- Moved form styling to a separate custom-style.css file
- Created contact.js for handling form submission and success message

### CODE REVIEW

**Review by sarahosterhed — COMMENTED** [review]
Good job with the contact form and implementing different options! 👏🏻

One thing I see though is that you implemented bootstrap styling on the header and footer. In the beginning of the project, Linn created a script for the header and footer that we were supposed to use and link to and I think we should keep this on all pages to ensure that the design of these elements always look the same for consistency.

File: pages/contact.html [hunk 1]
```diff
@@ -6,24 +6,80 @@
...
     <!-------------------- Fonts ------------------->
     <style>
         @import url('https://fonts.googleapis.com/css2?family=Inclusive+Sans:ital@0;1&display=swap');
     </style>
-
 </head>
 <body>
     <!-- Include header -->
-    <header>
+    <header class="d-flex justify-content-between align-items-center p-3">
```
Comment (linntarv) [inline pages/contact.html:20]: Use common header, by importing <script src="header.js"></script>

**Review by linntarv — COMMENTED** [review]
Looks good! Only thing is to import header from header.js so it looks the same on all pages

**Review by sarahosterhed — CHANGES_REQUESTED** [review]
Good job with the contact form and implementing different options! 👏🏻

We discussed this card during the sprint review and didn't close this task because of three main reasons:
- The header and footer should be implemented from the scripts like on all the other pages to maintain consistency across the site.
- There are some tags/symbols missing in the code, so it would be good to validate it before we merge it.
- You are using bootstrap for the styling, but this is something that we should have discussed when initializing the project, if we wanted to use a framework for the styling. Now, all the other pages are styled with regular css and for best practice we agreed that all pages should be styled in the same way.

**Review by Julieanna97 — COMMENTED** [review]
> We discussed this card during the sprint review and didn't close it because of three main reasons:
> - The header and footer should be implemented from the scripts like on all the other pages to maintain consistency across the site.
> - There are some tags/symbols missing in the code, so it would be good to validate it before we merge it.
> - You are using bootstrap for the styling, but this is something that we should have discussed when initializing the project, if we wanted to use a framework for the styling. Now, all the other pages are styled with regular css and for best practice we agreed that all pages should be styled in the same way.
> 
> These issues would need to be adressed. 

Noted. :) I will change it to CSS.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <link rel="stylesheet" href="css/contact.css"> <!-- Link to your new CSS file -->
+
+    <!-------------------- Fonts ------------------->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inclusive+Sans:ital@0;1&display=swap');
+    </style>
+</head>
+<body>
+    <!-- Load header dynamically -->
+    <script src="header.js"></script>
```
Comment (linntarv) [inline contact.html:19]: update file path, should be: scripts/header.js

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                <p>Vår adress:</p>
+                <p>1234 Main St<br>Stockholm, Sweden</p>
+                <p>Email: info@boklusen.se</p>
+                <p>Telefon: +46 123 456 789</p>
+            </div>
+        </div>
+    </div>
+
+    <!-- Load footer dynamically -->
+    <script src="footer.js"></script>
```
Comment (linntarv) [inline contact.html:59]: update these paths too: scripts/footer.js & scripts/contact.js

**Review by linntarv — DISMISSED** [review]
Looks great! Just remember to update the file paths to the js files that are now in the folder scripts :)

**Review by mjp414 — COMMENTED** [review]
Looks good! 
Just wondering if the footer is correct? It looks like the script is commented in the HTML-file and other coding has been added. Otherwise it looks good 🙂

**Comment by Julieanna97** [comment]
> Looks good! Just wondering if the footer is correct? It looks like the script is commented in the HTML-file and other coding has been added. Otherwise it looks good 🙂

Tried adding the script.js but the links are not clickable. So I added manually instead.

**Pull request merged**